src/audio/aria/tune/sof_aria_blobs.m

```diff
@@ -20,17 +20,16 @@ function sof_aria_blobs()
 
 	blob8 = sof_aria_build_blob(0);
 	tplg2_fn = sprintf("%s/passthrough.conf", sof_tplg_aria);
-	check_create_dir(tplg2_fn);
-	tplg2_write(tplg2_fn, blob8, "aria_config", ...
-		    "Exported with script sof_aria_blobs.m" , ...
-		    "cd tools/tune/aria; octave sof_aria_blobs.m");
+	sof_tplg2_write(tplg2_fn, blob8, "aria_config", ...
+			"Exported with script sof_aria_blobs.m" , ...
+			"cd tools/tune/aria; octave sof_aria_blobs.m");
```

Comment:
Hmm, check_create_dir() is dropped from this one place, but not all? I guess this call on old L23 was unnecessary and so can be removed? I'd prefer a rename (that touches so many places) to be in its own commit and put any small refactoring changes to separate commits.

Reply:
I don't remember exactly why so, but I'll try to split this. I try to keep every commit functional to not break bisect. There was probably some issue with those path create helpers.

Reply:
It was a lazy delete, there was no directory check in sof_check_create_dir(). It's now added to 2nd commit as well as these cleanups.